packages/dialtone-tokens/postcss/constants.cjs

```diff
@@ -9,7 +9,7 @@
  */
 
 module.exports = {
-  HSLA_EXCLUDED_COLORS: ['--dt-color-gradient-magenta-purple', '--dt-badge-color-background-ai', '--dt-color-border-ai'],
+  HSLA_EXCLUDED_COLORS: ['--dt-color-gradient-gold-red-magenta-purple', '--dt-color-gradient-magenta-purple', '--dt-badge-color-background-ai', '--dt-color-border-ai'],
```

Comment:
@braddialpad There's a new surface as well: 

![image](https://github.com/user-attachments/assets/646f379a-23b7-462b-bc78-1ebdb42cbafd)

Reply:
that should be fine, it's a semantic token so it just references the base token like so:

```
--dt-color-surface-ai: var(--dt-color-gradient-gold-red-magenta-purple);
```

Reply:
Nevermind, the one I posted above will work, but all the h/s/l ones won't because we aren't generating hsl for the gradient.

Reply:
added to excludes.